src/write.c

```diff
@@ -886,6 +886,55 @@ static int avifQualityToQuantizer(int quality, int minQuantizer, int maxQuantize
     return quantizer;
 }
 
+// Adds the items for a single cell or a grid of cells.
+// Outputs the mainItemId, being the only cell if there is exactly one, or the grid item for multiple cells.
+static avifResult avifEncoderAddImageItems(avifEncoder * encoder,
+                                           uint32_t gridCols,
+                                           uint32_t gridRows,
+                                           uint32_t gridWidth,
+                                           uint32_t gridHeight,
+                                           avifBool alpha,
+                                           uint16_t * mainItemId)
+{
+    const uint32_t cellCount = gridCols * gridRows;
+    const char * infeName = alpha ? "Alpha" : "Color";
```

Comment:
Nit: I suggest adding a local variable:

```
    size_t infeNameSize = 6;
```

and use it instead of the expression `strlen(infeName) + 1` below.

If you strongly prefer, you can call the `strlen()` function:

```
    size_t infeNameSize = strlen(infeName) + 1;
```

But I think 6 is clearly correct if it is right after the `const char * infeName = alpha ? "Alpha" : "Color";` line.